wiki/content/slash-graphql/slash-quick-start.md

```diff
@@ -2,16 +2,29 @@
 title = "Slash Quick Start"
 [menu.main]
     parent = "slash-graphql"
-    weight = 1   
+    weight = 1
 +++
 
-*These are draft docs for Slash GraphQL, which is currently in beta*
+## Introduction
 
-Welcome to Slash GraphQL. By now, you should have created your first deployment, and are looking for a schema to test out. Don't worry, we've got you covered.
```

Comment:
my only concern here is that it feels like I'm repeating myself in the next sentence:

`In this guide, we will create a database for a small app from the ground up using Slash GraphQL. The easy steps you will learn in this tutorial are fundamental to use Slash GraphQL, and mastering it will give you a better understanding of the powerful features available.`

Maybe mixing both sentences? what do you think?

(I know, probably our docs backlog is big enough to care about these "style concerns" & details)

Reply:
Looks great! Only nit is that I'd swap 'use' with 'using' in the second sentence.

Reply:
<please let me know if I should click "resolve conversation" at this point in the workflow :)>

Reply:
Alright, I'll commit the fix 👍 
`The easy steps you will learn in this tutorial are fundamental to using Slash GraphQL, and mastering it will give you a better understanding of the powerful features available.`